usagov/usagov-2021
# USAGOV-1927: Timeouts when updating English or Spanish menus

https://cm-jira.usa.gov/browse/USAGOV-1927

## Description
Increased the timeouts for the WAF, WWW, and CMS. Kept the WAF value higher than the others.
Added in a little utility to make a page wait/sleep in order to test timeout situations on dev/test (or I may remove this from the PR if this is not necessary).

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions
After this is deployed to Dev/Test we should (if we still have the wait-tool in place) be able to go to /site-lag-test?wait=90 which should take around 90 seconds to load. We can play around with the value, but if we go to a long wait-time like /site-lag-test?wait=130 then we should in theory get a Drupal-error page instead of a nging error-page.

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
